service/arm_templates/main.json

```diff
@@ -50,28 +50,93 @@
     "kubernetesIngressInsecurePort": {
       "type": "int"
     },
+    "masterCount": {
+      "type": "int",
+      "defaultValue": 1
+    },
+    "masterIPAddresses": {
+      "type": "array",
+      "metadata": {
+        "description": "An array containing all the master nodes' ip addresses."
+      }
+    },
+    "masterVMSize": {
+      "type": "string"
+    },
+    "masterDataDiskSizeGB": {
+      "type": "int"
+    },
+    "masterAdminUsername": {
+      "type": "string"
+    },
+    "masterAdminPassword": {
```

Comment:
I don't think password auth should be enabled. Can we remove the parameter and force the use of SSH keys?